Source/Core/InputCommon/InputConfig.cpp

```diff
@@ -58,10 +58,10 @@ bool InputConfig::LoadConfig(bool isGC)
 
     for (int i = 0; i < 4; i++)
     {
-      if (control_section->Exists(type + "Profile" + num[i]))
+      if (control_section->Exists(type + "Profile" + std::string(num[i])))
```

Comment:
Is this needed now that this type is `std::string_view`?

Reply:
gcc-9.2.0 throws a ton of errors about no valid operator+() overload if this isn't there.

Reply:
operator+ was specifically excluded from handling conversions to std::string in the standard (rationale being that part of the purpose of std::string_view was to avoid allocations, and if avoiding allocations is desirable, then allocating conversions should be made explicitly).